src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  background-color: black;
+}
+
+.outer,
+.inner,
+.content {
+  border: 1px solid #e95d49;
+  box-sizing: border-box;
+}
```

Comment:
Usually, there should be only one instance of certain selector per stylesheet.

For these styles you can create one more common class, like `block` and apply it to same HTML elements.